test/index.js

```diff
@@ -1 +1,5 @@
 require('./measures/measures-data-spec');
+require('./util/is-inverse-benchmark-spec');
+require('./util/format-decile-generator-spec');
+require('./util/format-csv-record-spec');
+require('./benchmarks/benchmarks-data-spec');
```

Comment:
Haven't quite decided if there will be tests there, but will clean up and remove if it remains empty.